src/persist/src/indexed/mod.rs

```diff
@@ -511,6 +514,11 @@ impl<L: Log, B: Blob> Indexed<L, B> {
                     )));
                 }
             }
+            Err(_) if TypeId::of::<B>() == TypeId::of::<UnreliableBlob<MemBlob>>() => {
```

Comment:
can we log these at a lower log level instead of totally removing the logs? could sometimes still be useful to understand test behavior?

Reply:
done